woocommerce/woocommerce-ios
# My Store Analytics: Add feature flag and new button for custom range

Closes: #11941 


## Description

This PR adds a new button for adding a custom range on My Store analytics. This button is hidden behind a feature flag so it should not be displayed on production yet.

## Testing instructions

- Enable the feature flag `customRangeInMyStoreAnalytics` and build the app.
- Notice on My Store there's a new button for adding a custom range. The tap action for this button has not been handled yet.
- Feel free to test again with the feature flag off to confirm that the button is not available.

## Screenshots

| Light mode | Dark mode |
| --- | --- |
| <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/fa97d400-15a0-4acd-95b4-3c0e7617a97f" width=320 /> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/b3f32652-3c51-4b79-a66d-79807c66d491" width=320 /> |

---
